### PR TITLE
NMS-182

### PR DESCRIPTION
**Pull request by srivastavaAbhi**
SaveCall detail may have only 1 element in Array.
Check has been updated and TC updated as per latest understanding.

JIRA issue : https://applab.atlassian.net/browse/NMS-182

### CODE REVIEW

File: testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java [hunk 1]
```diff
@@ -1274,6 +1274,8 @@ public void verifyFT85() throws IOException, InterruptedException {
     /**
      * To verify the that Save Inbox call Details API request should not succeed with content provided for only 1
      * subscription pack and not even a place holder for second Pack details
+     *
+     * //TODO : FT case to be modified .Refer NMS -182
```
Comment (frankhuster) [inline testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java]: Why not fix this as part of this pull request?

Comment (srivastavaAbhi) [inline testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java]: Code is modified, only thing left is Ft doc

Thanks,
Abhishek

Comment (frankhuster) [inline testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java]: OK, so I assume you will change the comment for this method (which currently doe snot match the code) when the FT description is updated?

Comment (srivastavaAbhi) [inline testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java]: FT description to be updated in next version of FT doc.
I have conveyed this to Kanchan.

**Pull request merged**